database-plugins/src/test/java/co/cask/hydrator/plugin/teradata/test/TeradataPluginTestRun.java

```diff
@@ -358,56 +358,22 @@ public void testNonExistentDBTable() throws Exception {
 
     ETLBatchConfig etlConfig = new ETLBatchConfig("* * * * *", sourceBadName, table, transforms);
     Id.Application appId = Id.Application.from(Id.Namespace.DEFAULT, "dbSourceTest");
```

Comment:
should change the name of this app.  There are travis test failures that seem to indicate some other test uses the same app name.